library/src/helpers/extractHashData.ts

```diff
@@ -0,0 +1,58 @@
+import { CONTAINER_LABELS } from '../constants';
+
+const CONTAINERS_REGEX = new RegExp(
+  `(.*?)--(${Object.values(CONTAINER_LABELS).join('|')})$`,
```

Comment:
extract part of this expression, as it's the same as in ITEMS_REGEX

Reply:
Which part?